exonum/src/node/state.rs

```diff
@@ -690,6 +722,11 @@ impl State {
         self.blocks.get(hash)
     }
 
+    /// Returns incomplete block with the specified hash.
+    pub fn incomplete_block(&self, hash: &Hash) -> Option<&IncompleteBlock> {
```

Comment:
Can there be several incomplete blocks? Do they correspond to different heights? (The answer seems to be no, given that the `incomplete_blocks` container is cleared on transition to a new height.) I thought that an incomplete block is pushed into the structure only if its `BlockResponse` is verified to be authorized by a supermajority of validators; i.e., multiple blocks indicate Byzantine collusion.